tests/queries/0_stateless/02493_analyzer_uniq_injective_functions_elimination.sql

```diff
@@ -1,5 +1,9 @@
-SET allow_experimental_analyzer = 1;
+SET allow_experimental_analyzer = 1, optimize_injective_functions_inside_uniq = 1;
 
 EXPLAIN QUERY TREE SELECT uniqCombined(tuple('')) FROM numbers(1);
 
 SELECT uniqCombined(tuple('')) FROM numbers(1);
+
+EXPLAIN QUERY TREE SELECT uniqCombined(tuple(materialize(tuple(number)))) FROM numbers(10);
+
+SELECT uniqCombined(tuple(materialize(tuple(number)))) FROM numbers(10);
```

Comment:
The lack of tests for this feature is extremely sad to me (not your fault, it was bad already). 0 surprises it did not work properly, and 0 surprises if it breaks again in the future.

Do you mind adding tests that check `non-injective` functions and multi-argument functions too, please? It'd be a really great addition.

Reply:
Thank you @canhld94